src/uu/users/src/users.rs

```diff
@@ -5,19 +5,97 @@
 
 // spell-checker:ignore (paths) wtmp
 
+use std::ffi::OsString;
+use std::path::Path;
+
 use clap::builder::ValueParser;
 use clap::{crate_version, Arg, Command};
+use uucore::error::UResult;
 use uucore::{format_usage, help_about, help_usage};
 
-mod platform;
+#[cfg(target_os = "openbsd")]
+use utmp_classic::{parse_from_path, UtmpEntry};
+#[cfg(not(target_os = "openbsd"))]
+use uucore::utmpx::{self, Utmpx};
 
 const ABOUT: &str = help_about!("users.md");
 const USAGE: &str = help_usage!("users.md");
 
 static ARG_FILES: &str = "files";
 
+fn get_long_usage() -> String {
+    #[cfg(not(target_os = "openbsd"))]
+    let default_path: &str = utmpx::DEFAULT_FILE;
+    #[cfg(target_os = "openbsd")]
+    let default_path: &str = "/var/log/wtmp";
```

Comment:
There is a mismatch with the default path used in `uumain`: `/var/run/utmp` (my guess is that this is the correct path). It might make sense to define an OpenBSD specific const with the default path and then use this const in both places.

Reply:
Thanks @cakebaker , fixed this based on your comment